pyteomics/proforma.py

```diff
@@ -2534,3 +2563,56 @@ def find_tags_by_id(self, tag_id, include_position=True):
     @property
     def tags(self):
         return [tag for tags_at in [pos[1] for pos in self if pos[1]] for tag in tags_at]
+
+    def composition(self, include_charge=False, aa_comp=None):
+        '''
+        Calculate the elemental composition of the ProForma sequence.
+
+        Parameters
+        ----------
+        include_charge : bool, optional
+            If True, the charge state will be included in the composition.
+            Otherwise, composition of the neutral molecule will be returned.
+            Defaults to False.
+        aa_comp : dict, optional
+            A dictionary mapping amino acid symbols to their respective
+            compositions. If not provided, the standard amino acid composition
+            will be used.
+
+        Returns
+        -------
+        Composition
+            :py:class:`Composition` object representing the composition of the ProForma sequence.
+        '''
+        comp = Composition()
+        if aa_comp is None:
+            aa_comp = std_aa_comp
+        try:
+            for aa, tags in self.sequence:
+                comp += aa_comp[aa]
+                for tag in tags or []:
+                    comp += tag.composition
+            for tag in chain(self.fixed_modifications, self.labile_modifications, self.unlocalized_modifications):
```

Comment:
`fixed_modifications` should be checked when visiting each amino acid, similar to this:
https://github.com/levitsky/pyteomics/blob/f4033cb67f842102c6362a3b19fb14c811b6f801/pyteomics/proforma.py#L2337-L2340

Reply:
Thanks, should be fixed now.